cvat/schema.yml

```diff
@@ -6097,6 +6705,9 @@ components:
         mode:
           type: string
           readOnly: true
+        frame_count:
```

Comment:
There should probably be documentation on how this relates to `start_frame` and `stop_frame`.

Reply:
This is the actual frame count in the job, what description would you like to add? Probably, I'd defer it till all fields are described.

Reply:
Why it must be equal to that number? There can also be skipped frames in the task configuration and only specific frames in the job segment. It's just the actual number of frames.

Reply:
Alright.